packages/governance/src/types-ambient.js

```diff
@@ -725,4 +725,6 @@
  * @property {LimitedCF<SF>} creatorFacet creator-like facet within the governed contract (without the powers the governor needs)
  * @property {GovernorCreatorFacet<SF>} governorCreatorFacet of the governing contract
  * @property {Awaited<ReturnType<SF>>['publicFacet']} publicFacet
+ * @property {Instance} instance
```

Comment:
this is of the governed contract. that unqualified naming matches `creatorFacet` (even though it's "limited") and `publicFacet` but it doesn't match unqualified `adminFacet` which is of the _governing_ contract.

I suppose these new names are good and separately `s/adminFacet/governorAdminFacet`, which is out of scope. I've made a note in https://github.com/Agoric/agoric-sdk/issues/7178